tests/integration/submissions/test_submissions_views.py

```diff
@@ -262,6 +261,8 @@ def test_new_literature_submit(app, api_client, create_user_and_token, requests_
         == history.headers["Authorization"]
     )
     assert history.url == f"{app.config['INSPIRE_NEXT_URL']}/workflows/literature"
+    assert "datetime" in post_data["data"]["acquisition_source"]
+    del post_data["data"]["acquisition_source"]["datetime"]
```

Comment:
why we delete that?

Reply:
https://github.com/inspirehep/inspirehep/blob/d3968f24b810a91ef1711daf4600976a7af6dcf7/tests/integration/records/serializers/test_latex.py#L11

Reply:
alternative would be to to mock `time` (not sure if we have it) and test MOCKED time. I don't think this has a lot of value compared to effort needed. So checking if it's there or not it's fair IMO.